docs/get-started/cardano-wallet-js.md

```diff
@@ -0,0 +1,805 @@
+---
+id: cardano-wallet-js
+title: Get Started with cardano-wallet-js
+sidebar_label: cardano-wallet-js
+description: Get Started with cardano-wallet-js
+image: ./img/og-developer-portal.png
+---
+
+# cardano-wallet-js
+`cardano-wallet-js` is a javascript/typescript SDK for Cardano with a several functionalities. You can use it as a client for the official [cardano-wallet](https://github.com/input-output-hk/cardano-wallet) and also to create Native Tokens and NFTs. 
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Usage](#usage)
+    + [Connecting to a cardano-wallet service](#connecting-to-a-cardano-wallet-service)
+    + [Blockchain Information](#blockchain-information)
+  * [Useful operations](#useful-operations)
+    + [Generate Recovery Phrases](#generate-recovery-phrases)
+    + [Create or restore a wallet](#create-or-restore-a-wallet)
+    + [Wallet addresses](#wallet-addresses)
+    + [Wallet balances](#wallet-balances)
+    + [Wallet delegation](#wallet-delegation)
+    + [Stake pool operations with the wallet](#stake-pool-operations-with-the-wallet)
+    + [Wallet transactions](#wallet-transactions)
+    + [Submit external transaction](#submit-external-transaction)
+    + [Key handling](#key-handling)
+    + [Native Tokens](#native-tokens)
+		+ [Send Native Tokens](#send-native-tokens)
+- [Test](#test)
+- [Support our project](#support-our-project)
+
+
+## Introduction
+The official cardano-wallet by IOHK exposes a REST api/CLI interface which allows
+clients to perform common tasks on the cardano-blockchain, such as:
+ - creating or restoring a wallet
+ - submitting a transaction with or without [metadata](https://github.com/input-output-hk/cardano-wallet/wiki/TxMetadata) 
+ - checking on the status of the node
+ - listing transactions
+ - listing wallets
+
+Our project aims to provide an easy to use Javascript SDK for programmers, instead of
+exposing the raw REST structure to you. 
+
+Finally, it helps you to build desktop wallet clients - like [Daedalus](https://daedaluswallet.io/) - with
+embedded cardano-wallet binaries, so you don't necessarily have to 
+connect to a remote cardano-wallet server.
+
+## Warning
```

Comment:
Can this be changed to important admonition , and moved above Introduction?

(PS: Will leave it to you if you want to include, but the use case for cardano-wallet are mostly custodial services and full node wallet management - specifying few clients can become slightly ambiguous)